app/controllers/api/v1/lobbies_controller.rb

```diff
@@ -3,19 +3,41 @@
 module Api
   module V1
     class LobbiesController < AuthenticatedController
-      before_action :require_authentication, except: [:from_code, :join]
+      before_action :require_authentication, except: [:from_code, :join, :answer]
+      before_action :set_lobby, only: [:join, :start, :answer, :show, :update, :destroy]
       before_action :require_authorisation, only: [:show, :update, :destroy]
       before_action :set_quiz, only: [:index, :create]
 
       def join
-        lobby = Lobby.find(params[:id])
+        @player = @lobby.players.create(player_params)
 
-        @player = lobby.players.create(player_params)
-
-        Pusher.trigger(lobby.code, Lobby::PLAYER_JOIN, { id: @player.id })
+        Pusher.trigger(@lobby.code, Lobby::PLAYER_JOIN, { id: @player.id })
         render :player, status: :created
       end
 
+      def start
+        @lobby.status = :in_progress
+        @lobby.save
```

Comment:
Another way to do this is `@lobby.update(status: :in_progress)`.

A better way to do it would be to use the bang (`!`) version of the method, so that in case the save/update fails, it throws an error 😄

The best way to do it would be to also handle that error in one way or another, but we can also rely on simply our endpoint returning a 500 for now.